src/nvim/os/input.c

```diff
@@ -482,7 +490,7 @@ static void input_read_cb(Stream *stream, RBuffer *buf, size_t c, void *data, bo
 
 static void process_interrupts(void)
```

Comment:
This function name is confusing. You have to read the impl to understand that it's actually setting `got_int`, not processing interrupts. 

```suggestion
static void process_ctrlc(void)
```